docs/architecture/009-use-standard-for-url-structure/0009-supporting-packageurl.md

```diff
@@ -0,0 +1,106 @@
+# Investigation of using PackageURL as Command 'url'/location standard
+
+## Description
```

Comment:
I think for this whole section it might be ok to just link to the purl-spec documentation